web/src/components/Testimonial/Testimonial.module.scss

```diff
@@ -122,6 +122,10 @@
   &__person {
     align-items: center;
     display: flex;
+
+    > picture {
+      flex-shrink: 0;
```

Comment:
Is there another way we can fix this issue? Ideally we wouldn't target element selectors directly. Think Shilpa had suggested switching hard-coded image heights for `height: auto` as a potential fix?